src/RA_Interface.h

```diff
@@ -40,81 +40,85 @@ enum EmulatorID
 enum ConsoleID
 {
     UnknownConsoleID = 0,
-    MegaDrive = 1,	//	DB
-    N64 = 2,
-    SNES = 3,
-    GB = 4,
-    GBA = 5,
-    GBC = 6,
-    NES = 7,
-    PCEngine = 8,
-    SegaCD = 9,
-    Sega32X = 10,
-    MasterSystem = 11,
-    PlayStation = 12,
-    Lynx = 13,
-    NeoGeoPocket = 14,
-    GameGear = 15,
-    GameCube = 16,
-    Jaguar = 17,
-    DS = 18,
-    WII = 19,
-    WIIU = 20,
-    PlayStation2 = 21,
-    Xbox = 22,
-    Events = 23, // not an actual console
-    XboxOne = 24,
-    Atari2600 = 25,
-    MSDOS = 26,
-    Arcade = 27,
-    VirtualBoy = 28,
-    MSX = 29,
-    C64 = 30,
-    ZX81 = 31,
-    //unused32 = 32,
-    SG1000 = 33,
-    VIC20 = 34,
-    Amiga = 35,
-    AmigaST = 36,
-    AmstradCPC = 37,
-    AppleII = 38,
-    Saturn = 39,
-    Dreamcast = 40,
-    PSP = 41,
-    CDi = 42,
-    ThreeDO = 43,
-    Colecovision = 44,
+    MegaDrive        = 1, //	DB
+    N64              = 2,
+    SNES             = 3,
+    GB               = 4,
+    GBA              = 5,
+    GBC              = 6,
+    NES              = 7,
+    PCEngine         = 8,
+    SegaCD           = 9,
+    Sega32X          = 10,
+    MasterSystem     = 11,
+    PlayStation      = 12,
+    Lynx             = 13,
+    NeoGeoPocket     = 14,
+    GameGear         = 15,
+    GameCube         = 16,
+    Jaguar           = 17,
+    DS               = 18,
+    WII              = 19,
+    WIIU             = 20,
+    PlayStation2     = 21,
+    Xbox             = 22,
+    Events           = 23, // not an actual console
+    XboxOne          = 24,
+    Atari2600        = 25,
+    MSDOS            = 26,
+    Arcade           = 27,
+    VirtualBoy       = 28,
+    MSX              = 29,
+    C64              = 30,
+    ZX81             = 31,
+    // unused32 = 32,
```

Comment:
this comment is confusing the aligner. another vote for disabling it.

Reply:
Sure